.mailmap

```diff
@@ -0,0 +1,95 @@
+Achal Shah <achals@gmail.com> <achal@uber.com>
+Adenilson Cavalcanti <cavalcantii@gmail.com> <a.cavalcanti@samsung.com>
+Adenilson Cavalcanti <cavalcantii@gmail.com> <a.cavalcanti@sisa.samsung.com>
+Adrian Heine <mail@adrianheine.de>
+Agoston Szepessy <agszepp@gmail.com>
+Alexander Popiak <alex@zenguard.org> <alexander.popiak@gmail.com>
+Alexandrov Sergey <splavgm@gmail.com>
+Antonio de Luna <tonydelun@gmail.com>
+Askeing Yen (fyen) <askeing@gmail.com>
+Attila Dusnoki <dati91@gmail.com> <dati91@users.noreply.github.com>
+bd339 <bd339@tuta.io> <bd339@localhost.localdomain>
+Beomjin Kim <beomjin.kim@lge.com> <memory128m@gmail.com>
+Bogdan Cuza <bogdan.cuza@hotmail.com> <boghison22@gmail.com>
+bors-servo <release+servo@mozilla.com> <lbergstrom+bors@mozilla.com>
+bors-servo <release+servo@mozilla.com> <metajack+bors@gmail.com>
+Brian Anderson <banderson@mozilla.com> <andersrb@gmail.com>
+Brian J. Burg <burg@cs.washington.edu>
+Clark Gaebel <cg.wowus.cg@gmail.com> <cgaebel@uwaterloo.ca>
+Clark Gaebel <cgaebel@mozilla.com> <cg.wowus.cg@gmail.com>
+Connor Brewster <connor.brewster@eagles.oc.edu> <brewsterc@my.caspercollege.edu>
+Daniel Robertson <dan.robertson@anidata.org> <danlrobertson89@gmail.com>
+Daniel Robertson <dan.robertson@anidata.org> <drobertson@tripwire.com>
+David Raifaizen <d-raif@hotmail.com> <whoknows>
+Dirkjan Ochtman <dirkjan@ochtman.nl> <d.ochtman@activevideo.com>
+edunham <edunham@mozilla.com> <github@edunham.net>
+Emilio Cobos Álvarez <me@emiliocobos.me> <ecoal95@gmail.com>
+Emilio Cobos Álvarez <me@emiliocobos.me> <emilio@crisal.io>
```

Comment:
Can you put emilio@crisal.io as primary email? I don't use the other that much :)